backend/python/Dockerfile

```diff
@@ -6,7 +6,6 @@ COPY requirements.txt ./
 RUN pip install -r requirements.txt
 
 COPY . ./app
-VOLUME file_uploads:/var/lib/file_uploads
```

Comment:
i think this double declaration in here and the compose file might have caused the file volume OCI runtime error. however i got the error pretty inconsistently before making this change so not sure if it's just coincidence that i haven't seen it again. removing it doesn't seem to cause any issues with running the containers though